build.gradle

```diff
@@ -1932,6 +1954,82 @@ project(':connect:basic-auth-extension') {
   }
 }
 
+project(':remote-storage-managers:hdfs') {
+  archivesBaseName = "kafka-rsm-hdfs"
+
+  configurations {
+    localDeps
+  }
+
+  dependencies {
+    localDeps group: 'org.apache.hadoop', name: 'hadoop-client', version: '3.1.2'
+    compile configurations.localDeps
+    compile project(':core')
+
+    testCompile libs.junit
+    testCompile group: 'org.apache.hadoop', name: 'hadoop-minicluster', version: '3.1.2'
+    testCompile project(':core')
+    testCompile project(':core').sourceSets.test.output
+    testCompile project(':clients').sourceSets.test.output
+    testRuntime libs.slf4jlog4j
+  }
+
+  checkstyle {
+    configProperties = checkstyleConfigProperties("import-control-remote-storage-managers.xml")
+  }
+
+  tasks.create(name: "copyDependantLibs", type: Copy) {
+    from(configurations.localDeps - project(':core').configurations.runtime - project(':tools').configurations.runtime) {
+      exclude('kafka-rsm-hdfs*')
+    }
+    into "$buildDir/dependant-libs"
+    duplicatesStrategy 'exclude'
+  }
+
+  jar {
+    dependsOn copyDependantLibs
+  }
+}
+
+project(':remote-storage-managers:s3') {
+  archivesBaseName = "kafka-rsm-s3"
+
+  configurations {
+    localDeps
+  }
+
+  dependencies {
+    compile group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.11.726'
```

Comment:
hello @satishd  I think you need to update 'compile' by 'localDeps' in order to generate "dependant-libs".